### PR TITLE
Add courses seed data

### LINKED ISSUE #3855: E-learning: Add seed data for Courses
30.times do
  Course.create!([{
    title: Faker::Educator.course_name,
    description: Faker::TvShows::GameOfThrones.quote
  }])
end

### PR DESCRIPTION
**Pull request by mattlindsey**
Add seed data for new Courses table/model.

fixes #3855

### CODE REVIEW

File: db/seeds.rb [hunk 1]
```diff
@@ -77,6 +77,20 @@ def get_country
     )
 
     puts 'Created default projects'
+
+    opensource = u.courses.create!(
```
Comment (faithngetich) [inline db/seeds.rb]: I see we have an object called opensource but I can't seem to find where this is referenced

File: db/seeds.rb [hunk 2]
```diff
@@ -77,6 +77,20 @@ def get_country
...
+      title: 'OpenSource',
+      description: 'How to contribute to open source projects.',
+      status: 'Active'
+    )
+
+    gitcourse = u.courses.create!(
```
Comment (faithngetich) [inline db/seeds.rb]: same as above. If we are not using those variables: `gitcourse` and `opensource`, we should consider removing them.

File: db/seeds.rb (hunk 1 above)
Comment (mattlindsey) [reply db/seeds.rb]: Fixed.

File: db/seeds.rb (hunk 2 above)
Comment (mattlindsey) [reply db/seeds.rb]: Fixed.

**Pull request merged**